mycroft/skills/skill_loader.py

```diff
@@ -278,46 +297,26 @@ def _load_skill_source(self):
             except Exception as e:
                 LOG.exception('Failed to load skill: '
                               '{} ({})'.format(self.skill_id, repr(e)))
-            else:
-                module_is_skill = (
-                    hasattr(skill_module, 'create_skill') and
-                    callable(skill_module.create_skill)
-                )
-                if module_is_skill:
-                    return skill_module
-        return None  # Module wasn't loaded
+        return skill_module
 
     def _create_skill_instance(self, skill_module):
-        """Use v2 skills framework to create the skill."""
+        """create the skill object.
+
+        Arguments:
+            skill_module (module): Module to load from
+
+        Returns:
+            (bool): True if skill was loaded successfully.
+        """
         try:
-            self.instance = skill_module.create_skill()
+            skill_class = get_skill_class(skill_module)
+            self.instance = skill_class()
```

Comment:
This should be tested with breakpoints and a watch, just in case (I'm on a phone)